bang/magicollections.py

```diff
@@ -790,8 +779,20 @@ def to_fields(self, item, extra_fields=None, exclude_fields=None, order=None, *a
                         'link_text': cameo.name,
                     } for cameo in item.cached_cameos]
                 }))
-            # Add live2d viewer
+            # Add live2d viewer and chibis
             if hasattr(item, 'associated_costume'):
+                chibis_set = item.associated_costume.owned_chibis.all()
+                if chibis_set:
+                    extra_fields.append(('chibis', {
+                        'icon': 'pictures',
+                        'type': 'images',
+                        'verbose_name': _('Chibi'),
+                        'images': [{
+                            'value': chibi.image_url,
+                            'verbose_name': _('Chibi'),
+                        } for chibi in chibis_set],
```

Comment:
```python
{
    ...
    'type': `images_links`,
    ...
    'images': [{
        'value': chibi.image_url,
        'link': chibi.original_image_url,
        'link_text': u'{} - {}'.format(unicode(item), _('Chibi')),
        'verbose_name': _('Chibi'),
    } for chibi in item.costume_chibis],
}
```

This way when users click on the chibi image they can download the full size, uncompressed image.
The link_text is purely for SEO.